src/main/java/com/pnudev/springonlinedelivery/Order.java

```diff
@@ -0,0 +1,33 @@
+package com.pnudev.springonlinedelivery;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.Getter;
+
+@Entity
+@Table(name = "'order'")	
```

Comment:
This is not the right type of character. You need backticks (`) instead of single-quotes (')